mis_builder/wizard/mis_builder_dashboard.py

```diff
@@ -71,11 +71,15 @@ def action_add_to_dashboard(self):
             arch = self.env["ir.ui.view.custom"].browse(last_customization[0].id).arch
         new_arch = etree.fromstring(arch)
         column = new_arch.xpath("//column")[0]
+        # Due to native dashboard doesn't support form view
+        # add "from_dashboard" to context to get correct views in "get_views"
+        context = self._context.copy()
+        context['from_dashboard'] = True
```

Comment:
```suggestion
        context = dict(self.env.context, from_dashboard=True)
```

to avoid using the deprecated `self._context`?